### ISSUE TITLE
[9.4] Lost password 

### ISSUE THREAD

**Issue by jcpreaux** [glpi-project/glpi#5386]
Hello,

In GLPI v9.4, lost password link sent by email doesn't work, an error "Your password reset request has expired or is invalid. Please renew it." is displayed on the front.

In sql-error.log, i've the following lines : 

glpisqllog.ERROR: DBmysql::query() in /var/sites/hotline/inc/dbmysql.class.php line 177
  *** MySQL query error:
  SQL: SELECT * FROM `glpi_users` WHERE `password_forget_token` = 'a529b6fd1b76862fca6b639c9de0be358e930ee2' AND NOW() < ADDDATE(password_forget_token_date, INTERVAL 1 DAY))
  Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1
  Backtrace :
  inc/dbmysqliterator.class.php:95
  inc/dbmysql.class.php:569                          DBmysqlIterator->execute()
  inc/user.class.php:4395                            DBmysql->request()
  front/lostpassword.php:52                          User::showPasswordForgetChangeForm()

Thx

JCP

**Describe the bug**

**Page(s) URL**

If applicable, page(s) URL where the bug happens.

**To reproduce**

**Expected behavior**

**Logs**

```
paste logs here
Find them in *-error.log files under glpi/files/_log/
```

**Screenshots**

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

```
paste here
```

**Additional context**

**Comment by cedric-anne** [comment]
Hi,

Could ou test fix provided in #5387 ?

Regards

**Comment by trasher** [comment]
Please do not close issues. It will be done automatically when PR will be merged.
